_inc/client/setup-wizard/feature-toggle/map-feature-to-props.js

```diff
@@ -0,0 +1,1028 @@
+/**
```

Comment:
I wonder if we shouldn't be pulling those descriptions from string we already have. I believe we can get them all from each module props.

Reply:
Most of the descriptions are unique as @jessefriedman made them for this project, and several of the features here don't have corresponding modules we could easily pull strings for, so I figured just having them all declared here would be best. We could try to share the declarations of some of the strings that already exist elsewhere in the code but I think that's best left for a different PR.